src/style.css

```
@@ -1,3 +1,85 @@
+html {
+  --header-height: 60px;
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding-top: 10px;
```

```suggestion
  padding: 0 50px;
```
why do you need top padding. by design should be horizontal paddings for header